unity-renderer-desktop/Assets/Scripts/MainScripts/DCL/Components/Video/AvPro/AvProVideoPlayer.cs

```diff
@@ -13,9 +14,11 @@ public class AvProVideoPlayer : IVideoPlayer, IDisposable
     private Texture avProTexture;
     private string lastError;
     private string id;
-    private bool resizeDone;
     private bool doLoop;
 
+    private CancellationTokenSource resizeVideoTextureCancellationSource;
```

Comment:
```suggestion
    private CancellationTokenSource resizeVideoTextureCts;
```

the `Cts` acronym is widely accepted for CancellationTokenSource. I leave it up to you if you prefer the long name.